pkg/auth/providers/local/local_auth_provider.go

```diff
@@ -109,17 +109,19 @@ func (l *Provider) AuthenticateUser(ctx context.Context, input interface{}) (v3.
 	username := localInput.Username
 	pwd := localInput.Password
 
+	authFailedError := httperror.NewAPIError(httperror.Unauthorized, "authentication failed")
 	user, err := l.getUser(username)
-
 	if err != nil {
 		// If the user don't exist the password is evaluated
 		// to avoid user enumeration via timing attack (time based side-channel).
 		bcrypt.CompareHashAndPassword(l.invalidHash, []byte(pwd))
-		return v3.Principal{}, nil, "", err
+		logrus.Debugf("Authentication failed: %v", err)
```

Comment:
Can we expand this to specify it's from the user get, having duplicate log messages makes it hard to tell exactly where something come from

Reply:
Updated log messages to be unique and included username in both.